test/config.custom.yaml

```diff
@@ -36,3 +36,10 @@ electricity:
 
   extendable_carriers:
     Link: [H2 pipeline]
+
+# clean_cooking configuration
+clean_cooking:
+  enable: true
+  fuel: ["AC", "lpg", "pellets", "firewood", "charcoal", "heat"]
+  p_nom: [1, 1, 1, 1, 1, 1] # same order as fuels
```

Comment:
What does this do? How would that be useful when more than 1 country is modelled?

Reply:
I only forgot to remove this from the test config, otherwise removed it everywhere else. It is not being used at all